Content.Shared/Xenoarchaeology/Artifact/SharedXenoArtifactSystem.XAT.cs

```diff
@@ -90,6 +90,13 @@ public void TriggerXenoArtifact(Entity<XenoArtifactComponent> ent, Entity<XenoAr
 
         if (node != null && unlockingComp.TriggeredNodeIndexes.Add(GetIndex(ent, node.Value)))
         {
+            // DeltaV - start of faster unlock effect
+            if (ent.Comp.UnlockCompleteDuration != null && TryGetNodeFromUnlockState((ent.Owner, unlockingComp, ent.Comp), out var unlockingNode))
+            {
+                unlockingComp.EndTime = _timing.CurTime + ent.Comp.UnlockCompleteDuration.Value;
```

Comment:
Change the `ent.Comp.UnlockCompleteDuration != null` to `ent.Comp.UnlockCompleteDuration is {} unlockDuration` (or whatever local variable you wanna set), and then you can use that variable in place of the `ent.Comp.UnlockCompleteDuration.Value`.